calMean.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
```

Comment:
Are you sure this file should be a part of your contribution?

If so, how about moving it to `misc/calc_mean.py`?

Another tip: make it so that the array to be sorted can be passed in as an argument or input to your program! 😃